crypto/mem_sec.c

```diff
@@ -266,7 +266,7 @@ static size_t sh_getlist(char *ptr)
     for (; bit; bit >>= 1, list--) {
         if (TESTBIT(sh.bittable, bit))
             break;
-        OPENSSL_assert((bit & 1) == 0);
+        assert((bit & 1) == 0);
```

Comment:
FWIW, my view is that these internal failures are only possible because of memory corruption, it cannot happen because of any kind of external (user) input.

I don't have a strong view on what to do when they fail.  But trying to keep running is doomed to failure.